src/main/resources/anniv6Resources/localization/eng/BrokenSpace/Relicstrings.json

```
@@ -0,0 +1,104 @@
+{
+  "${ModID}:BrokenDataDisk": {
+    "NAME": "Broken Data Disk",
+    "FLAVOR": "no",
+    "DESCRIPTIONS": [
+      "At the start of each combat, lose #b",
+      " #yFocus. The first time you channel an orb each turn, channel another orb of the same type."
+    ]
+  },
+  "${ModID}:BrokenSneckoSkull": {
+    "NAME": "Broken Snecko Skull",
+    "FLAVOR": "no",
+    "DESCRIPTIONS": [
+      "When you gain a buff, gain #b",
+      " more."
+    ]
+  },
+  "${ModID}:BrokenRedSkull": {
+    "NAME": "Broken Red Skull",
+    "FLAVOR": "no",
+    "DESCRIPTIONS": [
+      "When you are above #b",
+      "% #yHP, you have #b",
+      " extra #yDexterity."
+    ]
+  },
+  "${ModID}:BrokenOmamori": {
+    "NAME": "Broken Omamori",
+    "FLAVOR": "no",
+    "DESCRIPTIONS": [
+      "Add #b",
+      " #yCurses to your deck. Whenever you rest, upgrade #b",
+      " random cards in your deck."
+    ]
+  },
+  "${ModID}:BrokenDamaru": {
+    "NAME": "Broken Damaru",
+    "FLAVOR": "no",
+    "DESCRIPTIONS": [
+      "At the end of your turn, you may #yExhaust a card in your hand. Gain #yMantra equal to it's cost.",
+      "Select a card to exhaust for mantra."
```

Probably best to capitalize Mantra since it's a keyword (though card select screen like this are basically the wild west for text consistency).
```suggestion
      "Select a card to exhaust for Mantra."
```